client/pcr_test.go

```diff
@@ -87,6 +122,108 @@ func TestReadPCRs(t *testing.T) {
 	}
 }
 
+func CreatePCRSelection(s []int) ([]byte, error) {
```

Comment:
I think we need to leave this exported as it would be a useful tool for users outside of the client testing environment. Perhaps we should move it into a different file.

Reply:
If we aren't _currently_ using this outside of the testing environment. We should make a decision about exporting or not exporting this function in a followup PR (and leave it unexported for now).

Reply:
Resolved.